aptos-move/aptos-vm/src/sharded_block_executor/counters.rs

```diff
@@ -21,3 +24,31 @@ pub static SHARDED_BLOCK_EXECUTION_SECONDS: Lazy<HistogramVec> = Lazy::new(|| {
     )
     .unwrap()
 });
+
+/// Count of the committed transactions since last restart.
+pub static SHARDED_BLOCK_EXECUTOR_TXN_COUNT: Lazy<IntCounterVec> = Lazy::new(|| {
+    register_int_counter_vec!(
+        "sharded_block_executor_txn_count",
+        "Count of number of transactions per shard per round in sharded execution",
+        &["shard_id", "round_id"]
+    )
+    .unwrap()
+});
+
+pub static CROSS_SHARD_STATE_VALUE_WAIT_SECONDS: Lazy<HistogramVec> = Lazy::new(|| {
```

Comment:
how about just add a "name" label to avoid adding multiple similar timers? like https://github.com/aptos-labs/aptos-core/blob/11cfb78a1741df89b6d14f5bf9fed57626ae7f33/execution/executor/src/metrics.rs#L55

Reply:
Good point, done.